docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,482 @@
+  predictor:
+    model:
+      protocolVersion: v2
+      modelFormat:
+        name: openvino_ir
+      storageUri: "oci://localhost/ovms-resnet50:test"
+      args:
+      - --layout=NHWC:NCHW
+```
+
+Save the example above of the InferenceService to a file, and create it on the cluster:
+```sh
+oc create -n ai-demo -f ./FILE.yaml
+```
+
+Output:
+```
+inferenceservice.serving.kserve.io/ovms-resnet50 created
+```
+
+Soon, a Deployment and a Pod should appear in that namespace.
+Depending on the size of ServingRuntime's image and size of the ModelCar OCI image,
+it may take a while for the Pod to become ready.
+
+```sh
+$ oc get -n ai-demo deployment
+NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
+ovms-resnet50-predictor   1/1     1            1           72s
+```
+
+```sh
+$ oc get -n ai-demo pod
+NAME                                       READY   STATUS    RESTARTS      AGE
+ovms-resnet50-predictor-6fdb566b7f-bc9k5   2/2     Running   1 (72s ago)   74s
+```
+
+Kserve will also create a Service.
+
+```sh
+$ oc get svc -n ai-demo
+NAME                      TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+ovms-resnet50-predictor   ClusterIP   None         <none>        80/TCP    119s
+```
+
+#### Specifying hardware accelerators
+
+InferenceService can also include plethora of different options.
+For example it can contain `resources` section that is passed to the Deployment and then to the Pod,
+so the model server gets access to the NVIDIA hardware (thanks to the device plugin):
+```yaml
+spec:
+  predictor:
+    model:
+      resources:
+        limits:
+          nvidia.com/gpu: 1
+        requests:
+          nvidia.com/gpu: 1
+```
+
+For complete InferenceService specification refer to [kserve API Reference](https://kserve.github.io/website/latest/reference/api/).
+
+### Create Route
+
+> Note: You don't need to wait for model server Pod's readiness before creating Route.
+
+Finally, we can create an OpenShift Route CR to expose the service.
+You can either use `oc expose svc` command or create definition in a YAML file and apply it.
+
+```sh
+$ oc expose svc -n ai-demo ovms-resnet50-predictor
+route.route.openshift.io/ovms-resnet50-predictor exposed
+
+$ oc get route -n ai-demo
+NAME                      HOST                                               ADMITTED   SERVICE                   TLS
+ovms-resnet50-predictor   ovms-resnet50-predictor-ai-demo.apps.example.com   True       ovms-resnet50-predictor
+```
+
+### Querying the model server
+
+Finally, we'll check if our model is ready for inference and we'll reuse OVMS examples to test the inference.
+
+Get the IP of the MicroShift cluster and assign it to the `IP` variable.
+Use the HOST value of the Route created in previous step and assign to the `DOMAIN` variable.
+Next, run the following `curl` command.
+Alternatively, instead of using `--connect-to "${DOMAIN}::${IP}:"`,
+you can use real DNS, or add the IP and the domain to the `/etc/hosts` file.
+```sh
```

Comment:
so right here I think you're going to give me the real DNS way; but it's the curl command; perhaps just give the curl and then show the DNS as an optional step

Reply:
If you have DNS configured so that `ovms-resnet50-predictor-ai-demo.apps.example.com` points to the MicroShift cluster, the this is enough:
```
curl -i "ovms-resnet50-predictor-ai-demo.apps.example.com/v2/models/ovms-resnet50/ready"
```